### PR TITLE
Appeal into main

### PR DESCRIPTION
**Pull request by foobar-1337**

### CODE REVIEW

File: src/integration-test/java/se/sundsvall/casedata/apptest/ErrandResourceIT.java [hunk 1]
```diff
@@ -81,7 +81,8 @@ class ErrandResourceIT extends CustomAbstractAppTest {
 		"createdBy",
 		"updatedBy",
 		"note.*\\.createdBy",
-		"note.*\\.updatedBy"
+		"note.*\\.updatedBy",
+		"appeals" // TODO: Remove when service layer is implemented
```
Comment (ronnberg) [inline src/integration-test/java/se/sundsvall/casedata/apptest/ErrandResourceIT.java:85]: remove?

Comment (cheezi0747) [inline src/integration-test/java/se/sundsvall/casedata/apptest/ErrandResourceIT.java:85]: I assume service layer is implemented by now?

File: src/test/java/se/sundsvall/casedata/service/ErrandServiceTest.java [hunk 2]
```diff
@@ -372,55 +403,61 @@ void getDecisionsOnErrandNotFound() {
 	void deleteDecisionOnErrand() {
 		final var errand = toErrand(createErrandDTO());
 		errand.setCaseType(PARKING_PERMIT_RENEWAL.name());
-		final int sizeBeforeDelete = errand.getDecisions().size();
+		errand.getDecisions().size();
```
Comment (cheezi0747) [inline src/test/java/se/sundsvall/casedata/service/ErrandServiceTest.java:406]: This is not used, remove?

File: src/test/java/se/sundsvall/casedata/service/NoteServiceTest.java [hunk 3]
```diff
@@ -110,7 +107,7 @@ void testPatchNoteNotFound() {
 		final NoteDTO noteDTO = new NoteDTO();
 		final var problem = assertThrows(ThrowableProblem.class, () -> noteService.updateNote(1L, noteDTO));
 
-		assertEquals(Status.NOT_FOUND, problem.getStatus());
+		assertEquals(NOT_FOUND, problem.getStatus());
```
Comment (cheezi0747) [inline src/test/java/se/sundsvall/casedata/service/NoteServiceTest.java:110]: replace with assertThat

File: src/integration-test/resources/db/script/appealIT-testdata.sql [hunk 4]
```diff
@@ -0,0 +1,13 @@
+INSERT INTO errand (id, created,updated,version,application_received,case_title_addition,created_by,created_by_client,description,diary_number,end_date,errand_number,external_case_id,municipality_id,phase,priority,process_id,start_date,updated_by,updated_by_client,channel,case_type) VALUES
+	 (1, '2024-04-17 11:42:42.339545','2024-04-17 11:50:31.705008',4,'2024-04-17 11:38:47.49','Eldstad/rökkanal, Skylt','UNKNOWN','UNKNOWN','Some description of the case.','123','2022-06-01','BUILD-2024-000001','caa230c6-abb4-4592-ad9a-34e263c2787b','2281','Aktualisering','MEDIUM',NULL,'2022-01-01','UNKNOWN','UNKNOWN','EMAIL','NYBYGGNAD_ANSOKAN_OM_BYGGLOV');
+
+INSERT INTO stakeholder (id, created,updated,version,ad_account,authorized_signatory,first_name,last_name,organization_name,organization_number,person_id,`type`,errand_id) VALUES
```
Comment (ronnberg) [inline src/integration-test/resources/db/script/appealIT-testdata.sql:4]: Stakeholders is not shared between errand and decision. They are duplicated

File: src/integration-test/java/se/sundsvall/casedata/apptest/AppealIT.java [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+			.withServicePath(PATH + "/" + APPEAL_ID)
+			.withHeader(X_JWT_ASSERTION_HEADER_KEY, JWT_HEADER_VALUE)
+			.withHeader(AD_USER_HEADER_KEY, AD_USER)
+			.withHttpMethod(PATCH)
+			.withRequest(REQUEST_FILE)
+			.withExpectedResponseStatus(NO_CONTENT)
+			.withExpectedResponseBodyIsNull()
+			.sendRequestAndVerifyResponse();
+
+		final var updatedAppeal = appealRepository.findById(APPEAL_ID).orElseThrow();
```
Comment (ronnberg) [inline src/integration-test/java/se/sundsvall/casedata/apptest/AppealIT.java:70]: Why not read the entire errand (with setupCall) to match the entire structure after patch?

File: src/integration-test/java/se/sundsvall/casedata/apptest/AppealIT.java [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+import static se.sundsvall.casedata.service.util.Constants.X_JWT_ASSERTION_HEADER_KEY;
+
+@WireMockAppTestSuite(
+	files = "classpath:/AppealIT/",
+	classes = Application.class)
+@Sql({
+	"/db/script/truncate.sql",
+	"/db/script/appealIT-testdata.sql"
+})
+class AppealIT extends CustomAbstractAppTest {
```
Comment (ronnberg) [inline src/integration-test/java/se/sundsvall/casedata/apptest/AppealIT.java:33]: Skip usage of CustomAbstractAppTest. This should not be used.

**Review by cheezi0747 — APPROVED** [review]

**Pull request merged**